docs/development/release-process.md

```diff
@@ -77,3 +77,28 @@ Some description concerning major changes in this release, or potential incompat
 
 - #SOME_ID: @pull-request-creator Description
 ```
+
+## ArchLinux packages
+
+All of the information to build the ArchLinux package is in the AUR git repository (see [dokku AUR page](https://aur.archlinux.org/packages/dokku/)). The release of a AUR package only consists of pushing the package information into the AUR git repo. Then users could use that information to build the package on their machines.
+
+To update the package clone the repository and adjust the files in the repository. Then a helper script - `updpkgsums` - to update the SHA sum could  be called (check against the original SHA sum). Another helper script - `mksrcinfo` - needs to be called to update the meta information of the package in a file called `.SRCINFO`. The next step builds the package locally for verification - `makepkg`. As last step commit your changes and push the commit.
+
+* dependencies are defined in the `depends` attribute in `PKGBUILD`
+* build steps during package build time are defined in the `package()` method in `PKGBUILD`
+* steps that should be executed during install/update/remove time are defined in the file `dokku.install`
+* detailed information about all attributes in `PKGBUILD` could be found in the [ArchLinux wiki](https://wiki.archlinux.org/index.php/PKGBUILD)
+* detailed information about the AUR workflow could be found in the [AUR article](https://wiki.archlinux.org/index.php/Arch_User_Repository) in the ArchLinux wiki
+
+That is the usual workflow:
+
+```shell
+updpkgsums # update sha sums - compare them with the original ones
```

Comment:
> Can you update the Vagrantfile so I can have a vm for running these arch commands?

Where should the Vagrant file go? Because the build information is in a different repo (see below - the AUR repo).